src/components/add-mfa-method-sms/add-mfa-method-sms-routes.ts

```diff
@@ -0,0 +1,34 @@
+import * as express from "express";
+import { PATH_DATA } from "../../app.constants";
+import { requiresAuthMiddleware } from "../../middleware/requires-auth-middleware";
+import { validateStateMiddleware } from "../../middleware/validate-state-middleware";
+import {
+  addMfaAppMethodConfirmationGet,
+  addMfaSmsMethodGet,
+  addMfaSmsMethodPost,
+} from "./add-mfa-method-sms-controller";
+
+const router = express.Router();
+
+router.get(
+  PATH_DATA.ADD_MFA_METHOD_SMS.url,
+  requiresAuthMiddleware,
+  validateStateMiddleware,
+  addMfaSmsMethodGet
+);
+
+router.post(
+  PATH_DATA.ADD_MFA_METHOD_SMS.url,
+  requiresAuthMiddleware,
+  validateStateMiddleware,
+  addMfaSmsMethodPost
+);
+
+router.get(
```

Comment:
FYI We generally configure all the confirmation routes here: https://github.com/govuk-one-login/di-account-management-frontend/blob/main/src/components/update-confirmation/update-confirmation-routes.ts